server/portal/apps/notifications/routing.py

```diff
@@ -7,5 +7,5 @@
 from portal.apps.notifications.consumers import NotificationsConsumer
 
 websocket_urlpatterns = [
-    re_path(r'ws/notifications/$', NotificationsConsumer),
+    re_path(r'ws/notifications/$', NotificationsConsumer.as_asgi()),
```

Comment:
Related: https://channels.readthedocs.io/en/stable/releases/3.0.0.html#update-to-asgi-3